Automattic/woocommerce-payments
# Reduce Amount of Steps in the MOX

Fixes #8311

#### Changes proposed in this Pull Request

Note: This PR relates to the changes to the MOX identified in https://github.com/Automattic/woocommerce-payments/issues/8311. The scope of this PR is everything that happens **after** the Connect page/Jetpack setup, and **before** the redirection to the Stripe KYC. Connect page changes are addressed in #8378, and Jetpack changes will be addressed in a future piece of work.

* Reduces the amount of steps in the Onboarding MOX, removing the personal details and mode selection. (Noting that mode selection now happens earlier, implemented in #8378.)
* Styling updates to the form design.
* Adding the TOS note to the MOX. 









#### Screenshots

To follow

#### Testing instructions





* Spin up a new JN site from this branch using the `botwoo` below.
* Go to `Payments` and click `Finish Setup`.
* Connect Jetpack.
* Verify that the new onboarding flow looks like the designs (https://github.com/Automattic/woocommerce-payments/issues/8311).
* Test the form (e.g. trying to submit with inputs unselected, selecting then reselecting values and making sure the behaviour is as expected/consistent). 
* Continue to the Stripe KYC. Create an account and verify that it is successful, and that the values selected in the WooPayments MOX e.g. Country are either prefilled or not asked for again in the Stripe KYC.
* Test the flow again using a new JN site using mobile emulation in the browser and verify it works as expected.

-------------------


**Post merge**
